### DISCUSSION TITLE
Verification of student status cannot seem to use my location data

### DISCUSSION THREAD

**Discussion by Elliot-Fisk** [community/community#29875] [GitHub Education]
Hello, I am trying to verify my student status to get the various student benefits, but whenever I try to submit my student ID photo for review I get the following error. I have made sure that all the proper permissions are granted in chrome settings and performed multiple restarts of both chrome and my PC.
![image](https://user-images.githubusercontent.com/63561527/185262584-41252c7b-1f54-4bfa-ba0f-0e1ecf7d0192.png)
Any ideas? Thanks!

**Comment by nat246** [comment]
I'm also having the same problems and tried your methods.
I even tried verifying through my phone and I still get the same error.

**Comment by nat246** [comment]
Just tried it again, it works now. I didn't do anything just came back to it and tried it again after a few hours

**Comment by rishi-cmw** [comment]
I'm also having the same problem.. 😔

**Comment by Verkle1** [comment]
同

**Comment by n8fury** [comment]
Just reset your browser settings for github, and try again.
hope it works
